pinotdb/db.py

```diff
@@ -171,6 +171,9 @@ def close(self):
                 pass  # already closed
         # if we're managing the httpx session, attempt to close it
         if not self.is_session_external:
+            # TODO: This except clause has no point, as the session is always
+            #  provided by httpx and will never raise a pinotdb exception.
+            #  We should remove this.
```

Comment:
yeah I think we can just delete this. no need for a todo right?

Reply:
Sure, I just wanted to keep the mark there to reduce the scope of this particular PR. But if you want I can remove them right away.

Reply:
Fixed!